### PR TITLE
Redesign Add Source flow

### PR DESCRIPTION
**Pull request by gettinToasty**
Place existing sources higher in hierarchy so we discourage people from adding too many new sources
addsource

### CODE REVIEW

**Comment by holiber** [comment]
This toggle button is not correctly centered. The text baseline should be on the button's text level

File: test/filters.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ test('Adding and removing a Image Mask filter', async t => {
 
 test('Adding and removing a Crop Pad filter', async t => {
   const app = t.context.app;
-  const sourceName = 'Color Source';
+  const sourceName = 'Color Source 3';
```
Comment (holiber) [inline test/filters.js]: Why this change is needed?

**Comment by gettinToasty** [comment]
@holiber i fixed the toggle centering in the actual branch, the preview gif was just made before i did that

File: test/filters.js (hunk 1 above)
Comment (gettinToasty) [reply test/filters.js]: just easy logic to force the creation of a new source
